composer.json

```diff
@@ -30,6 +31,7 @@
         "doctrine/orm": "^2.9",
         "matthiasnoback/symfony-config-test": "^4.2",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
+        "phpoffice/phpspreadsheet": "^1.0",
```

Comment:
Can you add the lowest version to the `conflict` section

Reply:
@core23: The lowest version is 1.0 and I tested it with version 1.0.0. Is this still needed then?

Reply:
To me, no